packages/cozy-client/src/store/realtimes.js

```diff
@@ -1,6 +1,5 @@
 import { Mutations } from '../queries/dsl'
```

Comment:
This is really nitpicking, but why the plural in realtimes.js? There is only one, right?

Reply:
Yeah... I should change it. It was because other file were plurals in the folder...